benchmarks/dataset/flair/numpy.py

```diff
@@ -11,12 +12,15 @@
 
 from .common import get_label_mapping, get_multi_hot_targets, get_user_num_images
 
+logger = logging.getLogger(name=__name__)
+
 
 def make_federated_dataset(
         hdf5_path: str,
         partition: str,
         use_fine_grained_labels: bool,
         max_num_user_images: int,
+        scheduling_base_weight_multiplier: int = 1,
         numpy_to_tensor: Callable = lambda x: x) -> FederatedDataset:
     """
     Create federated dataset from the flair dataset, to use in simulations.
```

Comment:
This function now also controls the scheduling of users through a greedy algorithm. This is not alluded to in this docstring.